test/factories/featured_policies.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :featured_policy do
+    organisation
+    policy_content_id ''
```

Comment:
Factories should create valid objects, but this isn't properly valid with a blank content ID.